Task 1.1.4

AndreGGW opened the pull request:
Task 1.1.4 - ver-1

AndreGGW committed:
Task 1.1.4

pdasilem commented on src/main/java/jm/task/core/jdbc/Main.java:
```diff
@@ -0,0 +1,27 @@
+import jm.task.core.jdbc.service.UserServiceImpl;
+
+public class Main {
+    final static UserService userService = new UserServiceImpl();
```
не надо здесь статических полей

AndreGGW replied:
Сделал переменную локальной

pdasilem commented on src/main/java/jm/task/core/jdbc/dao/UserDaoHibernateImpl.java:
```diff
@@ -0,0 +1,116 @@
+            session.createSQLQuery("CREATE TABLE IF NOT EXISTS users" +
+                    "(id SERIAL PRIMARY KEY, name VARCHAR(255), last_name VARCHAR(255), age INT)").executeUpdate();
+            transaction.commit();
+        } catch (Exception e) {
```
сузь эксепшн до максимум хиберэксепшн

AndreGGW replied:
Exeption сузил до HibernateException.

pdasilem commented on src/main/java/jm/task/core/jdbc/service/UserServiceImpl.java:
```diff
@@ -0,0 +1,36 @@
+import java.util.List;
+
+public class UserServiceImpl implements UserService {
+    final private UserDao userDaoHibernate = new UserDaoHibernateImpl();
```
подумай, как лучше назвать эту переменную

AndreGGW replied:
Просто сделать ее UserDao userDao?
Поправил.
Поменял местами final и private, т.к. сперва должен стоять модификатор доступа.

AndreGGW committed:
Task 1.1.4

UserServiceImpl.java
final private UserDao userDaoHibernate -> private final UserDao userDao

UserDaoHibernateImpl.java
Exception e -> HibernateException e

Main.java
final static UserService userService -> внутрь psvm.
